help/schedule-a-message.md

```diff
@@ -87,6 +87,11 @@ can schedule a message for next morning to avoid disturbing others.
 1. Click the **trash** (<i class="fa fa-trash-o"></i>) icon on the message you
    want to delete.
 
+!!! keyboard_tip ""
+
+    You can also use <kbd>Delete</kbd> within the scheduled messages view to
```

Comment:
We want to use `Backspace` for the key here, so that it will update to `Delete` for Mac users and be `Backspace` otherwise.